lib/Quads.py

```diff
@@ -41,6 +42,19 @@ def __init__(self, config, statedir, movecommand, datearg, syncstate, initialize
         self.logger = logging.getLogger("quads.Quads")
         self.logger.setLevel(logging.DEBUG)
 
+        #EC528 addition - dynamically import driver module and set inventory and network services
+        inventoryservice = hardwareservice + "InventoryDriver"
+        networkservice = hardwareservice + "NetworkDriver"
+
+        importlib.import_module(inventoryservice)
+        importlib.import_module(networkservice)
+
```

Comment:
@portante another error was caused by importlib not being able to find the two modules (lines 49-50) in the path (we are using importlib.import_module here so that the specific driver modules can be imported dynamically, saving us from having to import all 6 modules every time). 

The working solution to this currently is to add their directories to the PYTHONPATH using:
sys.path.append(os.path.dirname(__file__) + "/hardware_services/inventory_drivers/")
sys.path.append(os.path.dirname(__file__) + "/hardware_services/network_drivers/")

which we put at the top of the file at the top of the file under the imports, and must have gotten lost in the shuffle (or added in a later branch). Not sure if this is the best way to go about it, so if you know of a better way, we can change it!